codec-stomp/src/test/java/io/netty/handler/codec/stomp/StompTestConstants.java

```diff
@@ -18,8 +18,8 @@
 public final class StompTestConstants {
     public static final String CONNECT_FRAME =
         "CONNECT\n" +
-            "host:stomp.github.org\n" +
             "accept-version:1.1,1.2\n" +
+            "host:stomp.github.org\n" +
```

Comment:
Fragile test I guess? The headers implementation before was keeping an overall insertion order [linked list](https://github.com/netty/netty/pull/3681/files#diff-c8a20821660fbcebfc60dea15a3a3f79L1351) which it used during all iterations.  Can you provide some background as to why that is no longer needed?  I think we are good for [HTTP](https://tools.ietf.org/html/rfc7230#section-3.2.2) and HTTP/2 we are also covered, but what about some of the others?